doc/api/cli.md

```diff
@@ -336,6 +343,64 @@ file will be created if it does not exist, and will be appended to if it does.
 If an error occurs while attempting to write the warning to the file, the
 warning will be written to stderr instead.
 
+### `--diagnostic-report-directory`
```

Comment:
Maybe all these sections should be placed between `--completion-bash` and `--enable-fips` sections alphabetically?